Makefile

```diff
@@ -25,6 +26,9 @@ DEFAULT_TARGETS = zpkg
 # and editing as appropriate.
 include CONFIG
 
+ifdef SDK_ROOT
+    BOOTGEN = $(SDK_ROOT)/bin/bootgen
+endif
```

Comment:
we can just call bootgen directly by sourcing the vivado shell script for whatever version we are using

Reply:
Thanks, I will investigate that possibility